lib/colorls/core.rb

```diff
@@ -1,3 +1,5 @@
+require 'clocale'
```

Comment:
Why is the require required here? ( No pun intended :wink: )

Reply:
It did not work without it... I probably would have to move that into `lib/colorls.rb` 😄

Reply:
Yep, moving the require should work.